### PR TITLE
doctest Canvas

### PR DESCRIPTION
**Pull request by chaosphere2112**
This pull request adds doctests to all of the examples in Canvas.py.

### CODE REVIEW

File: vcs/queries.py [hunk 1]
```diff
@@ -105,16 +104,18 @@ def graphicsmethodlist():
...
-    gm_list=a.graphicsmethodlist()
+            >>> a=vcs.init()
+            ...
 
-:returns: A list of available grapics methods (i.e., boxfill, isofill, isoline, outfill,
-          scatter, vector, xvsy, xyvsy, yxvsx, taylordiagram ).
-:rtype: list
-"""
+            >>> vcs.graphicsmethodlist() # Return graphics method list
+            ['boxfill', 'isofill', 'isoline', 'meshfill', 'scatter', 'vector', 'xvsy', 'xyvsy', 'yxvsx', ...]
```
Comment (doutriaux1) [inline vcs/queries.py]: all the yxvsx xyvsy etc... need to be replaced with 1d no?

Comment (doutriaux1) [inline vcs/queries.py]: also we need to add the 3d in here

File: vcs/queries.py [hunk 2]
```diff
@@ -123,31 +124,42 @@ def graphicsmethodtype(gobj):
...
+            >>> print vcs.graphicsmethodtype(iso)
+            isofill
+
+            >>> print vcs.graphicsmethodtype(ln)
+            Traceback (most recent call last):
+            ...
+            vcsError: The object passed is not a graphics method object.
+
+        :returns: If gobj is a graphics method object, returns its type: 'boxfill', 'isofill', 'isoline',
+                  'scatter', 'vector', 'xvsy', 'xyvsy', or 'yxvsx', 'taylordiagram'.
```
Comment (doutriaux1) [inline vcs/queries.py]: same here

File: vcs/queries.py (hunk 1 above)
Comment (embrown) [inline vcs/queries.py]: @doutriaux1 
Doctests have you supply a line of code of the format:
```python
>>> object.method()
'Return value'
```
When I call vcs.graphicsmethodlist() in a python console, it gives me the specified array.
The reason that I have the ```...``` at the end is that the array is too large to fit in 120 characters, and doctest will throw an error if I dont have the exact array, unless I throw ```...``` in there, which is kind of like a wildcard "match anything here" statement.
My thought was that the examples are to give a representation of what expected output should generally look like, not the exact details.
'xvsy', 'xyvsy', 'yxvsx', as well as '1d' and other things I don't show here show up in my array, when I call graphicsmethodlist().

File: vcs/queries.py (hunk 2 above)
Comment (embrown) [inline vcs/queries.py]: @doutriaux1 , I submitted https://github.com/UV-CDAT/vcs/pull/7 to address the issues you have with this PR.

**Pull request closed without merging**